lib/tooling/clang-tidy-tool.ts

```diff
@@ -41,14 +42,25 @@ export class ClangTidyTool extends BaseTool {
         this.addOptionsToToolArgs = false;
     }
 
-    async runTool(compilationInfo, inputFilepath, args, stdin, supportedLibraries) {
+    override async runTool(
+        compilationInfo: Record<any, any>,
+        inputFilepath: string,
+        args?: string[],
+        stdin?: string,
+        supportedLibraries?: Record<string, Library>,
+    ) {
         const sourcefile = inputFilepath;
         const options = compilationInfo.options;
         const dir = path.dirname(sourcefile);
-        const includeflags = super.getIncludeArguments(compilationInfo.libraries, supportedLibraries);
-        const libOptions = super.getLibraryOptions(compilationInfo.libraries, supportedLibraries);
+        const includeflags = supportedLibraries
```

Comment:
`compiler-dropin-tools.ts` does this as `const includeflags = this.getIncludeArguments(compilationInfo.libraries, supportedLibraries || {});` (and same with the getLibraryOptions call), they should be equivalent but we may as well try to be consistent here

Reply:
👍